wasip1/syscall_wasmedge_wasip1.go

```diff
@@ -260,6 +271,16 @@ func anyToSockaddr(rsa *rawSockaddrAny, port int) (sockaddr, error) {
 		addr := sockaddrInet6{port: port}
 		copy(addr.addr[:], rsa.addr[:])
 		return &addr, nil
+	case AF_UNIX:
+		addr := sockaddrUnix{}
+		n := 0
+		for ; n > len(rsa.addr); n++ {
+			if rsa.addr[n] == 0 {
+				break
+			}
+		}
```

Comment:
- the condition seems inverted here
- how about a shorter form?
```suggestion
		for n < len(rsa.addr) && rsa.addr[n] != 0 {
			n++
		}
```